File: desktop/release_channel.py
from __future__ import annotations
import os

def get_release_info() -> dict[str, object]:
    return {"product":"ZYRA AI","channel":os.environ.get("ZYRA_RELEASE_CHANNEL","beta"),"supported_channels":["beta","stable"],"installer_target":"win32-x64-nsis"}

File: electron/release_channel.js
const CHANNEL = process.env.ZYRA_RELEASE_CHANNEL || "beta";
function getReleaseInfo() { return { product:"ZYRA AI", channel:CHANNEL, supportedChannels:["beta","stable"], installerTarget:"win32-x64-nsis" }; }
module.exports = { getReleaseInfo };
